tests/test_etcd3.py

```diff
@@ -9,6 +9,7 @@
 import json
 import os
 import subprocess
+from urllib.parse import urlparse
```

Comment:
Could you use six.moves.urllib.parse for this? Otherwise it won't work with python 2.